### DISCUSSION TITLE
AutoProjectedGradientDescent

### DISCUSSION THREAD

**Discussion by khedher1984** [Trusted-AI/adversarial-robustness-toolbox#1690] [Q&A]
Hello,
I can't evaluate AutoProjectedGradientDescent.
Here is my configuration:
Tensorflow 2.4.3
Art 1.10
As I need to remove the last softmax layer, I apply this transformation:
atk_model = tf.keras.models.Model(inputs=model.input, outputs=model.get_layer(index=-2).output) 
Error: 
AttributeError: 'Tensor' object has no attribute 'numpy'

I have tried several proposed solutions but none of them work.
Thank you for your help.

**Comment by beat-buesser** [comment]
Hi @khedher1984 Can you provide more details?
